src/audio/kpb.c

```diff
@@ -58,7 +58,7 @@ struct comp_data {
 	bool is_internal_buffer_full;
 	size_t buffered_data;
 	struct dd draining_task_data;
-	size_t kpb_buffer_size;
+	size_t hb_buffer_size;
```

Comment:
care to comment these members, can be a subsequent PR.

Reply:
Sure, I will add another patch in the series for this PR.